internal/trigger/gaussian/gaussian_rate.go

```diff
@@ -183,3 +193,50 @@ func NewGaussianRateCalculator(peak time.Duration, stddev time.Duration, frequen
 		repeatWindow:  repeatWindow,
 	}
 }
+
+func calculateVolume(peakTps string, peakTime, stddev time.Duration) (float64, error) {
+	a, err := parseRateToTPS(peakTps) // the desired peak TPS
+	if err != nil {
+		return -1, err
+	}
+	b := peakTime.Seconds()
+	c := stddev.Seconds()
+
+	var total float64
+	for i := 0; i < 3600*24; i++ {
+		total += gauss(a, b, c, float64(i))
+	}
+
+	return math.Round(total), nil
+}
+
+func gauss(a, b, c, x float64) float64 {
+	return a * math.Exp(-(math.Pow(x-b, 2) / (2 * c * c)))
+}
+
+func parseRateToTPS(rateArg string) (float64, error) {
+	var rate int
+	var unit time.Duration
+	var err error
+	if strings.Contains(rateArg, "/") {
```

Comment:
I think this code is also duplicated here:

https://github.com/form3tech-oss/f1/blob/master/internal/trigger/constant/constant_rate.go#L58-L76

Perhaps it would be worth extracting it into its own function?